javav2/usecases/monitor_dynamodb/checkstyle.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+
+<!DOCTYPE module PUBLIC
```

Comment:
David stated that we should not check this into Github,

Reply:
Any reason?

Reply:
There is no real rule. If you think it makes sense, you can keep it.